testbenches/dramsim3.mk

```diff
@@ -126,10 +126,10 @@ $(TESTBENCH_PATH)/libdramsim3.so: $(BASEJUMP_STL_DIR)/bsg_test/bsg_dramsim3.cpp
 	$(CXX) $(CXXFLAGS) $(INCLUDES) $^ -Wl,-soname,$(notdir $@) -o $@
 
 endif # ifneq ($(filter $(_DRAMSIM3_MEM_CFGS), $(CL_MANYCORE_MEM_CFG)),)
-endif # ifndef(_BSG_F1_TESTBENCHES_DRAMSIM3_MK)
```

Comment:
is this undo-ing another PR?

Reply:
Yes/no. I was too aggressive in #526 and I moved it outside of the ifdef. This just fixes that.